syncstorage/db.go

```diff
@@ -217,23 +219,22 @@ func NewDB(path string, conf *Config) (*DB, error) {
   work is handled by private functions.
 */
 
-// LastModified gets the database modified time
-func (d *DB) LastModified() (modified int, err error) {
+// LastModified returns the top level last modified timestamp
+func (d *DB) LastModified() (int, error) {
 	d.Lock()
 	defer d.Unlock()
 
-	var m sql.NullInt64
+	lastMod, err := getKey(d.db, STORAGE_LAST_MODIFIED)
+	if lastMod == "" || err != nil {
+		return 0, err
```

Comment:
OK that's good to know that it won't totally corrupt your database when I deploy this. :) 
It seems that the python server also has a bug where deletion of a collection then a get doesn't increase the last modified at the whole storage level. :( Fortunately, that bug has been there for a while and it doesn't seem to have caused any major problems.